api/query/cache/index/filter.go

```diff
@@ -302,6 +309,27 @@ func (tr Triaged) Filter(t TestID) bool {
 	return false
 }
 
+// Filter interprets a TestLabel as a filter function over TestIDs.
+func (tl TestLabel) Filter(t TestID) bool {
+	name, _, err := tl.tests.GetName(t)
+	if err != nil {
+		return false
+	}
+
+	labels, ok := tl.metadata[name]
+	if !ok {
+		return false
+	}
+
+	for _, label := range labels {
+		if strings.ToLower(label) == strings.ToLower(tl.label) {
```

Comment:
Can you note that labels are case-insensitive in the docs? Or do labels need to be case-insensitive, does that follow a precedent of other query atoms?

Reply:
Added in the doc. I thought it would make more sense to make it case-insensitive in case of cap issues.

Reply:
I searched around a bit for other uses of `ToLower` or `ToUpper` to do case-insensitive compares, and found some:
https://github.com/web-platform-tests/wpt.fyi/blob/207813b3ed18bae81068934caa478daffd782d36/api/query/cache/index/filter.go#L166-L169
https://github.com/web-platform-tests/wpt.fyi/blob/207813b3ed18bae81068934caa478daffd782d36/api/query/search.go#L186
https://github.com/web-platform-tests/wpt.fyi/blob/207813b3ed18bae81068934caa478daffd782d36/api/query/atoms.go#L586

So there's some precedent for case-insensitive, so let's go with that! Thanks for documenting it!

Reply:
Thanks for double checking :)